index.html

```diff
@@ -9,6 +9,35 @@
 </head>
 
 <body>
+    <header>
+        <nav class="navegation">
+            <div class="logo">
+                <img src="images/logo 1.svg" alt="logo">
+            </div>
+
+            <div class="navegation-header">                    
+                <a href="">Home</a>
+                <a href="">About</a>
+                <a href="">Tech Atack</a>
+                <a href="">Projects</a>
+                <a href="">Contact</a>
+                <img src="images/akar-icons_github-fill.svg" alt="logo-guthub">
+                <img src="images/ant-design_twitter-circle-filled.svg" alt="logo-twitter">
+                <img src="images/entypo-social_linkedin-with-circle.svg" alt="logo-linkedin">
```

Comment:
Eu envolveria essas imagens em tags `a` pq no final das contas elas vão ser links para as suas redes sociais.

```suggestion
                <a href=""><img src="images/akar-icons_github-fill.svg" alt="logo-guthub"></a>
                <a href=""><img src="images/ant-design_twitter-circle-filled.svg" alt="logo-twitter"></a>
                <a href=""><img src="images/entypo-social_linkedin-with-circle.svg" alt="logo-linkedin"></a>
```